src/main/java/io/fabric8/maven/docker/util/WaitUtil.java

```diff
@@ -25,6 +25,8 @@
  */
 public class WaitUtil {
 
+    private static final int DEFAULT_DOCKER_TIMEOUT = 10;
```

Comment:
10s maybe a bit high as default, which is even used when no killGracePeriod is specified. In fact I wouldn't wait this no killGracePeriod is specified so simply skipping any wait when `int wait` is null.

Any particular reason why you always want a wait when a shutdown happens ?